app/src/main/res/values-is/strings.xml

```diff
@@ -334,9 +334,9 @@
                         <b>Virkjaðu skráadeilingu á Windows</b>\n
                     </li>
                     <p style='margin-left: 40px'>
-                        Opnaðu stjórnskjáinn (Control Panel), smelltu á 'Choose homegroup and sharing 
```

Comment:
You seem to have deleted a few "á" here.

Reply:
Fixed this.